packages/graphile-build-pg/src/QueryBuilder.d.ts

```diff
@@ -25,7 +25,7 @@ export default class QueryBuilder {
   public where(exprGen: SQLGen): void;
   public whereBound(exprGen: SQLGen, isLower: boolean): void;
   public setOrderIsUnique(): void;
-  public orderBy(exprGen: SQLGen, ascending: boolean): void;
+  public orderBy(exprGen: SQLGen, ascending: boolean, nullsFirst: boolean): void;
```

Comment:
Technically this should be `nullsFirst?: boolean` because it doesn't have to be specified

Reply:
You know what... Lets have it required.